bitbots_body_behavior/bitbots_body_behavior/decisions/reached_goal.py

```diff
@@ -40,23 +40,22 @@ def get_reevaluate(self):
         return True
 
 
-class AlignedToMoveBaseGoal(AbstractDecisionElement):
+class AlignedToGoal(AbstractDecisionElement):
```

Comment:
Also add path planning here, as this might get confusing in the dsd file if somebody thinks this decision checks if we are aligned to the soccer goal.

Reply:
Please rename this into aligned to path planning goal, as we have another aligned to goal decision which checks if the robot is aligned to the opponents soccer goal. Now a namespace conflict exist.

Reply:
@val-ba